bridge/functions.php

```diff
@@ -10,25 +10,37 @@
 function dd_tracing_enabled()
 {
     if ('cli' === PHP_SAPI) {
-        $cliEnabled = getenv('DD_TRACE_CLI_ENABLED');
-        if (false === $cliEnabled) {
-            return false;
-        }
-        $cliEnabled = strtolower(trim($cliEnabled));
-        return 'true' === $cliEnabled || '1' === $cliEnabled;
+        return dd_env_as_boolean('DD_TRACE_CLI_ENABLED', false);
     }
 
-    $value = getenv('DD_TRACE_ENABLED');
-    if (false === $value) {
-        // Not setting the env means we default to enabled.
-        return true;
+    return dd_env_as_boolean('DD_TRACE_ENABLED', true);
+}
+
+/**
+ * Returns the boolean value of an environment variable:
+ *  - if NOT defined then returns $default
+ *  - if defined and equals (case-insensitive) to 'true' or '1' then returns true
+ *  - if defined and equals (case-insensitive) to 'false' or '0' then returns false
+ *  - otherwise returns $default
+ *
+ * @param string $name
+ * @param boolean $default
+ * @return bool
+ */
+function dd_env_as_boolean($name, $default)
```

Comment:
Don't we have this same function somewhere else? Like somewhere in `Configuration::`?

Reply:
Correct, but in this file (bootstrap phase) we do not want to load anyone of our classes. Otherwise they might conflict with the ones composer might try to load.